languages/rust/exercises/concept/entry-api/tests/entry_api.rs

```diff
@@ -0,0 +1,36 @@
+use entry_api::*;
+
+#[test]
+fn test_example_works() {
+    let magazine = "two times three is not four".split_whitespace().collect::<Vec<&str>>();
+    let note = "two times two is four".split_whitespace().collect::<Vec<&str>>();
+    assert_eq!(can_construct_note(&magazine, &note), false);
+}
+
+#[test]
+#[ignore]
+fn test_fn_returns_true_for_good_input() {
+    let magazine = "The metro orchestra unveiled its new grand piano today. Its donor paraphrased Nathn Hale: \"I only regret that I have but one to give \"".split_whitespace().collect::<Vec<&str>>();
+    let note = "give one grand today.".split_whitespace().collect::<Vec<&str>>();
+    assert!(can_construct_note(&magazine, &note));
```

Comment:
```suggestion
    assert_eq!(can_construct_note(&magazine, &note), true);
```

While it's semantically identical and somewhat more typing, `assert_eq` gives more useful output when a test fails than `assert` does. This note applies to all subsequent tests also.